carbon_cassandra_plugin/carbon_cassandra_db.py

```diff
@@ -333,7 +333,9 @@ def _genAllWithWildcards():
 
       def _get(query):
         try:
-          cols = self.cfCache.get(cfName).get(query)
+          # Pycassa defaults to returning 100 columns without an explicit limit.
+          # This unreasonably large value is a HACK to let us fetch columns without having to page.
+          cols = self.cfCache.get(cfName).get(query, column_count=1999999999)
```

Comment:
we should use a loop and page here to avoid putting extra overhead on the cassandra servers to return the larger dataset